plugins/filter_tensorflow/tensorflow.h

```diff
@@ -0,0 +1,27 @@
+#ifndef FLB_FILTER_TENSORFLOW_H
```

Comment:
this file is missing the license header: https://github.com/fluent/fluent-bit/blob/master/CONTRIBUTING.md#licensing